tests/test_dictdiffer.py

```diff
@@ -208,8 +208,8 @@ def test_remove_list(self):
         assert ('remove', 'a', [(1, 'c'), (0, 'b'), ]) == diffed
 
     def test_add_set(self):
-        first = {'a': set([1, 2, 3])}
-        second = {'a': set([0, 1, 2, 3])}
+        first = {'a': {1, 2, 3}}
```

Comment:
we can change it only after officially removing Python 2.6 support.

Reply:
oki my bad

Reply:
Sorry, you can keep it. We have removed the 2.6 some time ago. I have rechecked `setup.py`.